airframe-tablet/src/main/scala/wvlet/airframe/tablet/text/ArrayJSONCodec.scala

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package wvlet.airframe.tablet.text
```

Comment:
I think airframe-codec will be a right repository for this code. 

We should move both ObjectJSONCodec and this ArrayJSONCodec to airframe-codec later. This is not necessary for this PR, but we need to add a Kaizen ticket for this.